libs/platform/src/lib/components/form/radio-group/radio-group.component.spec.ts

```diff
@@ -183,7 +183,7 @@ describe('GroupRadioButtonComponent', () => {
         await wait(fixture);
         fixture.detectChanges();
 
-        const fdradiobuttonsElem = fixture.debugElement.queryAll(By.css('fd-radio-button'));
+        const fdradiobuttonsElem = fixture.debugElement.queryAll(By.css('fdp-radio-button'));
```

Comment:
I dont think you need debug element here.

Reply:
In this PR, I'm just trying to fix the tests so they pass. I don't want to change the RadioGroup tests at this time. My main goal is to add unit test for FormGroup and FormField, fixing ActionBar and RadioGroup were just necessary steps to get everything passing.